alves/style-editor.css

```diff
@@ -1546,6 +1546,12 @@ p:not(.site-title) a:hover {
 	color: #394d55;
 }
 
+@counter-style empty {
+	.fse-template-part .main-navigation {
```

Comment:
I don't get why this is being prefixed with `.fse-template-part`.

Reply:
Ah, this is because in the editor CSS we wrap everything with the .fse-template-part selector.